challenge.rb

```diff
@@ -1,26 +1,26 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  input.map { |x| x.capitalize }
```

Comment:
looks good. take a peek at the & to_proc. you'll often see a common shorthand in ruby for example this could be written `input.map(&:capitalize)`

https://stackoverflow.com/questions/14881125/what-does-to-proc-method-mean